app/views/layouts/application.html.erb

```diff
@@ -21,15 +21,15 @@
     </div>
 
     <ul class="usa-accordion sidenav-list">
-      <% if @current_user.admin? %>
+      <% if @current_user && @current_user.admin? %>
```

Comment:
maybe we can create a method called `signed_in_as_admin?` that does both these checks